src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/BoxDataSourceItemFixture.cs

```diff
@@ -0,0 +1,57 @@
+using Reveal.Sdk.Dom.Core.Extensions;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class BoxDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_ValidDataSource()
+        {
+            // Arrange
+            string title = "Test Item";
+            var dataSource = new BoxDataSource();
+            var item = new BoxDataSourceItem(title, dataSource);
```

Comment:
the constructor should be in `// Act` section. And don't need to create the actual variables, just assert them directly to make the function shorter.